.github/actions/clang_format/action.yml

```diff
@@ -13,6 +13,14 @@ inputs:
 runs:
   using: "composite"
   steps:    
+      - name: Dependencies for local execution
+        if: env.ACT # Only run for local execution
+        shell: bash
+        run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+          echo "deb http://deb.debian.org/debian/ testing main" >>  /etc/apt/sources.conf
+          sudo apt update
```

Comment:
Question: This is for installing `clang-format`?

Reply:
It is